### ISSUE TITLE
Rule Proposal: Recommend against using Bootstrap classes inside CSS selectors

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#15]
Bootstrap classes are usually layout-oriented and don't contain any "data"-specific information about an element.

Example of a violation:

``` javascript
element.all(by.css("div.col-sm-offset-6 a"));
```

Though, I'm not completely sure this is a good fit for this specific plugin.

**Comment by alecxe** [comment]
Closed by 1f8dad682dce80b2a39ce0eebfea01bf8a3d8a48.
